assets/src/components/detours/detourMap.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react"
 import { LatLngLiteral } from "leaflet"
-import { Polyline, useMapEvents } from "react-leaflet"
+import { Polyline, Tooltip, useMapEvents } from "react-leaflet"
```

Comment:
**issue:** I see - this is actually a Leaflet tooltip, not a Bootstrap tooltip. I think the idea is that we wanted to use Bootstrap, not just have a style that visually matches Bootstrap. I could very well have missed some conversation last week, though.

Reply:
Granted, using Bootstrap / React-Bootstrap does have potential hurdles when it comes to the interactions with Leaflet on both the JavaScript and CSS fronts, but I'd like to at least think it through as an option.

Reply:
So @firestack and I paired for a little bit last week to try to get this to be a Bootstrap tooltip, and we couldn't really figure it out. From looking through the bootstrap docs, I couldn't even figure out how to attach a boostrap tooltip to a Polyline and have it actually show up, and even trying to style the Leaflet tooltip using bootstrap styles didn't work (the width was getting compressed for reasons that neither Kayla nor I was able to figure out). 

Do you have any idea how to go about that? Maybe I missed something simple in the bootstrap or react-bootstrap docs?

And also, isn't it an implementation detail whether we use Leaflet tooltips versus Bootstrap tooltips under the hood?

Reply:
Okay, if you all spent that much time on it and couldn't figure something out then I believe you. I just ask because the original ticket specifically said "Bootstrap tooltip."

> And also, isn't it an implementation detail whether we use Leaflet tooltips versus Bootstrap tooltips under the hood?

Sort of - from a purely user-facing functionality perspective, yes. However from a tooling perspective, both dev and design get certain benefits if it's a real Bootstrap component in terms of obeying certain application-wide styles and updating if they update. That said as we've established the "real Bootsrap" approach probably isn't technically feasible here, so it's fine to take another route.

Reply:
Do we get some of those benefits by pulling in the bootstrap variables the way I did in the CSS?

And/Or is there any way to get some or all of those benefits anyway?